MapboxDirections/MBRouteStep.swift

```diff
@@ -506,10 +506,18 @@ public class RouteStep: NSObject {
      */
     public let transportType: TransportType?
     
+    /**
+     Destinations, such as [control cities](https://en.wikipedia.org/wiki/Control_city), that appear on guide signage for the road identified in the `name` property.
+     
+     This property is typically available in steps leading to or from a freeway or expressway.
+     */
+    public let destinations: String?
```

Comment:
Developers will expect this property to be an array if we name it `destinations`, plural. I think it could be called `destination`, by analogy with `name`, which is singular despite the fact that it can sometimes contain multiple names in one string.

One thing I'm unsure about is whether it's a problem that "destination" already conceptually refers to the destination waypoint. But let's go with `destination` and revisit if there's any confusion. The documentation is pretty clear in any case.